beacon_node/lighthouse_network/src/peer_manager/mod.rs

```diff
@@ -502,6 +502,8 @@ impl<TSpec: EthSpec> PeerManager<TSpec> {
                     Protocol::BlocksByRange => PeerAction::MidToleranceError,
                     Protocol::BlocksByRoot => PeerAction::MidToleranceError,
                     Protocol::LightClientBootstrap => PeerAction::LowToleranceError,
+                    Protocol::LightClientOptimisticUpdate => PeerAction::LowToleranceError,
```

Comment:
Yes you are correct, so should we just return instead of `PeerAction::LowToleranceError` or something else? I fixed the other stuff other than merging in realbigsean's branch and adding the case for this the new rpcs. That should be easy but I've heard that it will make it harder to review if I rebase which I'll need to do in order to merge.

Reply:
Yeah I don't think we should ban them in this case (although it's an unexpected scenario) - I'll make it return and add a comment. This may change in the future if we decide to support running an actual light client using Lighthouse.